db/schema.rb

```diff
@@ -29,6 +29,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.integer  "affiliate_id"
+    t.string   "phone_number"
+    t.date     "dob"
```

Comment:
@xingyif we discussed during our meeting yesterday that Roberta wants date of birth included for Coordinators.  I named it 'dob' because that's how it is named for Students.